### ISSUE TITLE
Display Email Subject also for Automated Reply

### ISSUE THREAD

**Issue by jderecho** [chatwoot/chatwoot#7925]
### Is your feature or enhancement related to a problem? Please describe.

When an automated reply is sent out, no subject is displayed in the chat list preview. This makes it difficult to understand what the chat thread is about.

We need to enhance the system to ensure that the original email subject remains visible even when an automated reply is sent from Chatwoot. 

### Describe the solution you'd like

Ensure that when the automated reply is displayed, the original email subject is visible.

Ensure that the enhancement doesn't interfere with the readability and formatting of the email thread.

We propose to prepend the original chat subject before the last message content and concatenate it with a hyphen "-".
When a chat thread doesn't have a mail subject, ensure that the chat thread preview doesn't display the hyphen.

![image](https://github.com/chatwoot/chatwoot/assets/8219614/13bda18f-9c7b-4d14-9598-0e4b12237088)

**Comment by Evoltstore** [comment]
Yeah, it's a nightmare to follow up on messages when the subject of the email gets hidden

**Comment by sojan-official** [comment]
Note: Chatwoot will display the subject when present in the conversation list.  This case can occur only when the reply message would be missing the subject in its payload.  We will need to verify whether the automations are generating messages with subject appended in its content attributes.

**Comment by adriaandotcom** [comment]
I can't see any subject when sending emails to my inbox. Should I enable this somewhere?

**Comment by jesseallhands** [comment]
Related: Why is the subject not editable when composing an email response in a conversation? It often makes sense to edit/adjust the subject of an email response; there are a multitude of use-cases for this.
